File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<Integer, Integer>();
        int res = 0, maxCount = 0;
        for(int num : nums){
            map.put(num, map.getOrDefault(num,  1) + 1);
            if(map.get(num) > maxCount){
                res = num;
                maxCount = map.get(num);
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/sort-array-by-parity/submission-0.cs
public class Solution {
    public int[] SortArrayByParity(int[] nums) {
        for(int i = 0; i < nums.Length / 2; i++){
            if(nums[i] % 2 != 0 && nums[nums.Length - 1 - i] % 2 == 0){
                int temp = nums[i];
                nums[i] = nums[nums.Length - 1 - i];
                nums[nums.Length - 1- i] = temp;
            }
        }
        return nums;
    }
}